src/main/java/org/reaktivity/nukleus/sse/internal/stream/ServerStreamFactory.java

```diff
@@ -773,6 +793,46 @@ private void handleReset(
             final long traceId = reset.trace();
             doReset(applicationReplyThrottle, applicationRouteId, applicationReplyId, traceId);
         }
+
+        // TODO - SseEventFW
+        //                  .type = "challenge"
+        //                  .data = "{}"
+        // Received signalEx CHALLENGE from OAuth
+        private void handleSignal(
+            SignalFW signal)
+        {
+            final HttpSignalExFW httpSignalEx = httpSignalExRO.tryWrap(signal.buffer(),
+                                                                       signal.extension().offset(),
+                                                                       signal.extension().limit());
```

Comment:
Change to `final HttpSignalExFW httpSignalEx = signal.extension().get(HttpSignalExFW::tryWrap);`